### ISSUE TITLE
 Image features CNN(img)

### ISSUE THREAD

**Issue by nooralahzadeh** [ysmiura/ifcc#3]
Hi,

Would you please describe what steps (transform, pad, resize, ...) have you applied to provide CNN image features?

Thanks,

**Comment by ysmiura** [comment]
A random rotation, a color jittering, a padding to a square resolution, a resizing to 224x224, and a tensor normalization are applied to images. These image transformation processes can be found in [clinicgen/data/image2text.py](https://github.com/ysmiura/ifcc/blob/master/clinicgen/data/image2text.py#L48-L76).

**Comment by nooralahzadeh** [comment]
Thanks for the reply, May I ask you whether this transformation process has been applied in the same manner on the `Validation` and `Test` dataset as the `Train` dataset?

**Comment by ysmiura** [comment]
The random rotation and the color jittering are only applied to training data.

**Comment by nooralahzadeh** [comment]
Thanks for the reply,
May I ask you about the value for the `max-word`  that you use for the generation? is it as default `=128` or `=50`?

**Comment by ysmiura** [comment]
For the experiment in the paper, we used `max-word=128`.
